packages/backend/src/helpers/csp.ts

```diff
@@ -11,6 +11,7 @@ const helmetOptions: HelmetOptions = {
       connectSrc: [
         "'self'",
         // For Datadog RUM
+        'https://browser-intake-datadoghq.com',
         'https://*.browser-intake-datadoghq.com',
```

Comment:
Do we still need this?

Reply:
Not a priority to resolve now btw, just wondering. we can test another time.

Reply:
yes we actually need this, they changed the hostname for RUM